code/modules/mob/new_player/new_player.dm

```diff
@@ -525,7 +521,7 @@
 		.manifest tr.requested_department td {background-color: #00FF00}
 		.manifest th.reqhead td {background-color: #844}
 		.manifest tr.reqalt td {background-color: #FCC}
-		</style></head><body><center>Round Duration: [round(hours)]h [round(mins)]m<br>"}
```

Comment:
Undocumented text change

Reply:
Not nearly significant enough for a changelog entry.